app/Http/Controllers/ImageController.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Image;
+use Illuminate\Http\Request;
+use Illuminate\Support\Str;
+
+class ImageController extends Controller {
+    public function show(Request $request, $id) {
+        $image = Image::find($id);
+        if (!$image) abort(404);
+        $flag = $request->query('flag');
+
+        return response($flag === 's' ? $image->content_small : $image->content)
+            ->header('Content-Type', $image->mime_type)
+            ->header('Last-Modified', $image->created_at->toRfc7231String());
+    }
+
+    public function create(Request $request) {
+        if (!$request->hasFile('content'))abort(400, 'file is not uploaded');
+        $file = $request->file('content');
+        $mine_type = $file->getMimeType();
```

Comment:
~あなたのタイプには興味ないです~

```suggestion
        $mime_type = $file->getMimeType();
```
(後のもよしなに変えてね…)